app/controllers/api/v1/games_controller.rb

```diff
@@ -1,15 +1,10 @@
 class Api::V1::GamesController < ApplicationController
-  before_action :param_validation
-
   def show
     game = RawgFacade.get_game(params[:id])
-    render json: game, status: 200
-  end
-
-  private
-  def param_validation
-    if !params[:id].present?
-      render json: { data: {} }, status:400
+    if !game.nil?
+      render json: game, status: 200
+    else
+      render json: { description: "Error: No game found" }, status: 404
```

Comment:
little rails sugar here but this is a good option for readability

`status: :not_found`